pages/docs/getting-started.en-US.mdx

```diff
@@ -66,6 +66,20 @@ function useUser (id) {
 }
 ```
 
+Optionally, you can also leverage the `isValidating` property representing whether is fetching data or not.
+
+```jsx
+function useUser (id) {
+  const { data, error, isValidating } = useSWR(`/api/user/${id}`, fetcher)
+
+  return {
+    user: data,
+    isLoading: isValidating,
```

Comment:
My concern is that this might not be accurate. But the following is just kinda redundant..

```suggestion
    isLoading: isValidating && typeof user === 'undefined',
```

Reply:
Because `isValidating` can also mean “there is data, but SWR is revalidating due to refocus”, but “isLoading” feels like the initial fetching (where people need to display a spinner). Maybe it’s worth to introduce a  `status` field in SWR that contains enums of all possible states?

Reply:
Added the condition of checking data.
For `status`: I think it's depending on the case: wanna show the indicator for all the time or not 🤔 not very confident so far

Reply:
(This is more of discussion)

Yeah, my point was that the term "loading" feels blurry. When people say that something is loading, it usually means that a resource is turning from none existing to existing. 

And using `isValidating && !data` is kinda unnecessary: in most cases when `!data`, it **must** be validating so it's the same as `isLoading: !data`.

Ultimately I think the following terms are more precise:

- `isFetching`: is the fetching function being executed
- `isReady`: is the data ready to be used

Here `isFetching === isValidating` and `isReady === typeof data !== 'undefined'`, but less confusing.

Maybe we should have these built into SWR, wdyt?